src/mobility/getStationsById/index.ts

```diff
@@ -0,0 +1,30 @@
+import { RequestOptions } from '../../http'
+import { getServiceConfig, ArgumentConfig } from '../../config'
+import { mobilityQuery } from '../../api'
+
+import { Station } from '../types'
+
+import getStationsQuery from './query'
+
+export interface GetStationsByIdParams {
+    /** Return only stations with the given IDs. */
+    stationIds: string[]
```

Comment:
Bruk gjerne navnet `ids`, så blir det heilt likt som i APIet.

Reply:
Det er egentlig et bevisst valg for at det skal være utvetydig, siden det er en del ID'er å holde styr på, og `ids` da kan bety flere ting. 
F.eks. kan man tolke det som at man henter stasjoner tilhørende en operatørID eller systemID (hvis man ikke leser dokumentasjonen). Men hvis det er viktigere å ha det likt som i APIet så endrer vi.